src/rust/ide/src/model/suggestion_database/example.rs

```diff
@@ -0,0 +1,93 @@
+//! A module with structures related to examples found in Suggestion Database.
+use crate::prelude::*;
+
+use crate::double_representation::definition::DefinitionName;
+use crate::double_representation::module;
+use crate::double_representation::definition;
+
+use parser::Parser;
+
+
+
+// ==============
+// === Errors ===
+// ==============
+
+#[allow(missing_docs)]
+#[derive(Clone,Copy,Debug,Fail)]
+#[fail(display = "Invalid example code.")]
+pub struct InvalidExample;
+
+
+
+// ===============
+// === Example ===
+// ===============
+
+/// Example is a titled piece of code user can put into their graph to see and learn how to use
+/// our language.
+///
+/// If user picks the example, the `code` should became a body of a new method defined in current
+/// module. On scene the call for this method should appear.
```

Comment:
on scene -> on the scene
Plus, what is "the call for this method"? First of all the correct sentence would be "call OF this method", second of all, we dont have calls on scene, we have nodes, so this should be written as "On the scene, a node calling this function should appear."